rpm/packpack.spec

```diff
@@ -11,7 +11,11 @@ BuildRequires: sed
 BuildRequires: make
 Requires: bash
 Requires: coreutils
+%if (0%{?fedora} >= 31) || (0%{?rhel} >= 8)
+Requires: podman-docker
```

Comment:
Should not we keep ability to use docker: `podman-docker or docker`?

Reply:
There is no `docker` RPM on later RHEL/Fedora, it's `podman`. The `podman-docker` RPM gives you a compatibility layer `docker` executable so the rest of all the scripts still work. If it wasn't an `%else` it would fail.

Reply:
Ah, missed it. Thanks for the clarification!